aptos-move/framework/cached-packages/src/aptos_stdlib.rs

```diff
@@ -19,3 +19,27 @@ pub fn encode_test_coin_transfer(to: AccountAddress, amount: u64) -> Transaction
         vec![bcs::to_bytes(&to).unwrap(), bcs::to_bytes(&amount).unwrap()],
     ))
 }
+
+pub fn encode_create_resource_account(
+    seed: &str,
+    authentication_key: Option<AccountAddress>,
+) -> TransactionPayload {
+    let seed: Vec<u8> = bcs::to_bytes(seed).unwrap();
+    let authentication_key: Vec<u8> = if let Some(key) = authentication_key {
+        bcs::to_bytes(&key).unwrap()
+    } else {
+        vec![]
+    };
+    TransactionPayload::ScriptFunction(ScriptFunction::new(
+        ModuleId::new(
+            AccountAddress::ONE,
+            ident_str!("ResourceAccount").to_owned(),
+        ),
+        ident_str!("create_resource_account").to_owned(),
+        vec![],
+        vec![
+            bcs::to_bytes(&seed).unwrap(),
+            bcs::to_bytes(&authentication_key).unwrap(),
+        ],
+    ))
+}
```

Comment:
this shouldn't be needed any more

Reply:
Sorry, what is not needed? You meant the function `encode_create_resource_account`?

Reply:
Yeah, the same function is available at https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/cached-packages/src/aptos_sdk_builder.rs#L967

specifically
`aptos_stdlib::account_resource_create_resource_account`

Reply:
I fixed the generation of the entry point encoding just the other day :)